### DISCUSSION TITLE
FileIO - 让ClipCC项目获得访问本地文件的能力

### DISCUSSION THREAD

**Discussion by Flancok** [Clipteam/clipcc-gui#49] [扩展发布/Extensions sharing]
扩展英文名称：FileIO
扩展中文名称：FileIO
扩展介绍：FileIO是一个可以让ClipCC项目访问本地文件的扩展
开源地址：https://github.com/frank-782/clipcc-extension-fileio
下载地址：https://github.com/frank-782/clipcc-extension-fileio/releases/download/v1.1.0/fileio.extension@1.1.0.ccx 
**注意事项** 该扩展需要CC离线版才能运行，您可以前往ClipCC官方交流群959825608下载离线版使用
tags: `offline-required` `blocks-only` `open-source`  
![image](https://user-images.githubusercontent.com/46376077/120923060-0f43e500-c6ff-11eb-8729-72745c450dce.png)

**Comment by SimonShiki** [comment]
建议把使用前提贴一下 毕竟cc还没官方离线版

**Reply by Flancok** [reply]
加上了

**Comment by SimonShiki** [comment]
ClipCC扩展API已发生变更，请及时适配至最新API。

**Reply by Flancok** [reply]
新API的文档？另外，桌面端似乎还没有更新新API（

**Comment by SimonShiki** [comment]
又得更新了（悲
